app/experimenter/reporting/constants.py

```diff
@@ -0,0 +1,63 @@
+from django.db import models
+
+from experimenter.experiments.constants.legacy import ExperimentConstants
+from experimenter.experiments.constants.nimbus import NimbusConstants
+
+
+class ReportLogConstants(object):
+    EXPERIMENT_STATUS_CHOICES = tuple(
+        sorted(
+            list(
+                set(
+                    tuple(NimbusConstants.Status.choices)
+                    + ExperimentConstants.STATUS_CHOICES
+                )
+            )
+        )
+    )
```

Comment:
Yeah you could do it this way, but also with constants sometimes it's also fine to just explicitly refer them individually so it's a bit clearer 🤷‍♀️